src/pages/registration/description_tab/projects_field/ProjectItem.tsx

```diff
@@ -0,0 +1,127 @@
+        display: 'grid',
+        gridTemplateColumns: { sm: '1fr', md: '1fr 1fr auto' },
+        gap: '1rem',
+        bgcolor: 'secondary.light',
+        p: '1rem',
+        border: '1px solid lightgray',
+        borderRadius: '8px',
+      }}>
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+        <Typography fontWeight="bold">{t('project.project').toUpperCase()}</Typography>
+        <Box sx={{ display: 'grid', gridTemplateRows: '1fr auto', gap: '1rem', height: '100%' }}>
+          <div>
+            <Typography fontWeight="bold">{t('common.title')}:</Typography>
+            <Box sx={{ display: 'flex', gap: '2rem' }}>
+              {isFetching ? (
+                <StyledListSkeleton />
+              ) : project ? (
+                <MuiLink
+                  sx={{ display: 'flex', gap: '1rem', alignItems: 'center' }}
+                  component={Link}
+                  data-testid={dataTestId.registrationWizard.description.projectLink(project?.id)}
+                  to={getProjectPath(project.id)}
+                  target="_blank"
+                  rel="noopener noreferrer">
+                  {project.title}
+                  <OpenInNewIcon fontSize="small" />
+                </MuiLink>
+              ) : (
+                <Typography>-</Typography>
+              )}
+            </Box>
+          </div>
+          <div>
+            <Typography fontWeight="bold">{t('project.coordinating_institution')}:</Typography>
+            {isFetching ? (
+              <StyledListSkeleton />
+            ) : project?.coordinatingInstitution ? (
+              <Typography>{getLanguageString(project.coordinatingInstitution.labels)}</Typography>
+            ) : (
+              <Typography>-</Typography>
+            )}
+          </div>
+        </Box>
+      </Box>
+
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+        <Typography fontWeight="bold">{t('common.funding').toUpperCase()}</Typography>
+        {project?.funding?.length ? (
+          project.funding.map((funding) => (
+            <Box
+              key={funding.identifier}
+              sx={{ display: 'grid', gridTemplateRows: '1fr auto', gap: '1rem', height: '100%' }}>
+              <div>
+                <Typography fontWeight="bold">{t('registration.description.funding.funder')}:</Typography>
+                {isFetching ? <StyledListSkeleton /> : <Typography>{getLanguageString(funding.labels)}</Typography>}
+              </div>
+              <div>
+                <Typography fontWeight="bold">{t('project.grant_id')}:</Typography>
+                {isFetching ? (
+                  <StyledListSkeleton />
+                ) : funding.identifier ? (
+                  fundingSourceIsNfr(funding.source) ? (
+                    <Box sx={{ display: 'flex', gap: '2rem', height: 'fit-content' }}>
+                      <MuiLink
+                        sx={{ display: 'flex', gap: '1rem' }}
+                        component={Link}
```

Comment:
Som vi snakket om tidligere i dag så er `react-router` for å linke internt i vår app, så her holder det med `MuiLink`.
Husk alltid å sjekk at linkene (og annet virker også):
![bilde](https://github.com/user-attachments/assets/4ba20b65-66e2-4cc6-bbc8-ec144de550f5)

```suggestion
```